salt/metalk8s/repo/installed.sls

```diff
@@ -17,12 +17,16 @@
 {%- set image_fullname = docker_repository ~ '/' ~ image_name ~ ':' ~ image_version %}
 
 include:
+  - metalk8s.container-engine.containerd
```

Comment:
I do not really like this, it means if you configure repo (for example iso-manager) you will potentially upgrade (or downgrade) containerd as well.

I would suggest  to create a state to ensure containerd pkg is installed instead

Reply:
You're supposed to use `iso-manager` of the *current* version anyway, so the version of `containerd` on said system should be the one we want already, meaning executing this state should not result in any changes, no?

Reply:
Not exactly, for registry we have one file per version and we need to run states the state for all version deployed (especially the "new version" state so that we have the file for the new registry)